pkg/daemon/rpm-ostree.go

```diff
@@ -196,6 +196,48 @@ func podmanInspect(imgURL string) (imgdata *imageInspection, err error) {
 
 }
 
+// skopeoInspect is a wrapper around the "skopeo inspect" command. Skopeo gets the proxy
+// from our environment, reads /etc/containers/registries.* so it should be mirror-aware
```

Comment:
not that I'm doubting the mirror and proxy thing, but is there something we can use to verify?

Reply:
I haven't looked to through skopeo's tests to see what they have. 

I'd probably need a proxy and a registry to test it properly, I might be able to spin up a small one of each as part of a test. 

I could test it _improperly_ via a unit test though by just checking its output to see if it did a proxy connect when a proxy was supplied and a URL rewrite if registries.conf/.d specified it. 

We have a periodic that does proxy tests (periodic-ci-openshift-release-master-nightly-4.12-e2e-aws-ovn-proxy) I wonder if there's one that does ICSP/registry mirror stuff. There has to be prior art here somewhere, right?

Reply:
hmm, let me see if there is any proxy payload job somewhere